src/style.css

```diff
@@ -1,3 +1,52 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
```

Comment:
Avoid fixing container sizes like `width: 100%;` on the `header` if it's not a specific requirement. This can lead to overflow or accidental scroll bars if the content exceeds the container size. Let the content dictate the size of the container instead.